### DISCUSSION TITLE
Running statamic on railway.app?

### DISCUSSION THREAD

**Discussion by hleinan** [statamic/cms#9540] [Troubleshooting]
I have been using 3 days to get Statamic running om railway.app. Whatever I do, I am stoppet at permisjons in the cp (file_put_contents(xxx); Failed to open stream. Permission denied.) I have tried making a new volumet for storage, but no difference. I have tried working with Docker files, but that adds other issues. Anyone who has had success and can share some insight?

**Comment by duncanmcclean** [comment]
Forgive me, I don't know too much about Railway.app specifically. However, it *sounds* like it has an ephemeral filesystem which means you can't write to the filesystem, only read. 

Laravel Vapor has an ephemeral filesystem and we generally recommend folks use the [Eloquent Driver](https://github.com/statamic/eloquent-driver) for storing content, etc instead.
